src/js/oncoprint.js

```diff
@@ -0,0 +1,125 @@
+var Track = require('./track').Track;
+var _ = require('underscore');
+var d3 = require('d3');
+var $ = require('jquery');
+module.exports = {};
+
+// TODO: use self everywhere
+
+var defaultOncoprintConfig = {
+	cell_width: 10,
+	cell_padding: 3,
+	render: 'table',
+};
+
+function Oncoprint(container_selector_string, config) {
+	var self = this;
```

Comment:
I think it's a matter of taste.

Some benefits of this approach which we're not currently making use of but could potentially at some point are discussed briefly [http://stackoverflow.com/a/3658673/3158208](here)